### PR TITLE
[Docs]: updated GKE and EKS pages

### LINKED ISSUE #1172: [Docs] Update GKE and EKS Quick Start Instructions with `mesheryctl system config`
**Current State:**
`mesheryctl system config` is a new feature to assist in the configuration of the kubeconfig file used by Meshery for interacting with remote, managed Kubernetes clusters operated by Cloud providers.

**Desired State:**
Given this new feature, let's:

1. Take much of what is on the [GKE](https://meshery.layer5.io/docs/installation/gke#quick-start-with-google-kubernetes-engine-gke) and [EKS](https://meshery.layer5.io/docs/installation/eks#quick-start-with-google-kubernetes-engine-eks) pages and move these instructions into a "Manual Configuration" section. 
1. Above the "Manual Configuration" section, regurgitate the description of the `mesheryctl system config` function and its use.
1. Do this on both the GKE and EKS pages.

---
**Contributor Resources**
- [Meshery documentation site](https://meshery.layer5.io/docs/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

_Originally posted by @leecalcote in https://github.com/layer5io/meshery/pull/1170#issuecomment-648472944_

### PR DESCRIPTION
**Pull request by chandrashritii**
Signed-off-by: Shriti <shritichan@gmail.com>

This PR fixes #1172 

#### Checklist

- [x] [Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin).
- [x] PR title includes `[<name of area>]:<description>`

### CODE REVIEW

File: docs/pages/installation/eks.md [hunk 1]
```diff
@@ -6,7 +6,18 @@ permalink: installation/eks
 
 # Quick Start with Amazon Elastic Kubernetes Service (EKS)
 
-## Managed Kubernetes
+Make your way to Meshery and download the token by clicking the "Get Token" option in the dropdown menu. Utilize the token to run the following command:
```
Comment (leecalcote) [inline docs/pages/installation/eks.md:9]: I wonder if we can use a moniker for describing where to download the token, like "and download the token by clicking the 'Get Token' option under your User Account avatar."

File: docs/pages/installation/eks.md [hunk 2]
```diff
@@ -6,7 +6,18 @@ permalink: installation/eks
...
+
+| command           | flag                | function                                                     | Usage                     |
+|:------------------|:-------------------:|:-------------------------------------------------------------|:--------------------------|
+|                   | --system config     | configures Meshery with the kubeconfig, generated with the help of user details, to provide cluster access for public clouds(EKS). | `mesheryctl system config gke --token "PATH TO TOKEN"` |
+
```
Comment (leecalcote) [inline docs/pages/installation/eks.md:14]: I wonder if we would do well to include a final step of "Once configured, continue with the steps outlined in the Quick Start Guide (hyperlink) for your respective platform."

**Comment by chandrashritii** [comment]
updated :) please review @leecalcote

File: docs/pages/installation/eks.md [hunk 3]
```diff
@@ -6,7 +6,20 @@ permalink: installation/eks
 
 # Quick Start with Amazon Elastic Kubernetes Service (EKS)
 
-## Managed Kubernetes
+Make your way to Meshery and download the token by clicking the "Get Token" option in the dropdown menu under your User Account avatar.
```
Comment (kushthedude) [inline docs/pages/installation/eks.md:9]: This is a little ambiguous, can this be made a little more detailed like start meshery-ui, log in through the provider, go to `localhost:3000` & then download the token?

File: docs/pages/installation/gke.md [hunk 4]
```diff
@@ -5,19 +5,31 @@ permalink: installation/gke
 ---
 
 # Quick Start with Google Kubernetes Engine (GKE)
-You may perform the steps outlined under [Managed Kubernetes](#managedk8s) following by hand or run the [generate_kubeconfig_gke.sh](./generate_kubeconfig_gke.sh) shell script using the desired ServiceAccount name and Namespace arguments, like so:
 
-`./generate_kubeconfig_gke.sh cluster-admin-sa-gke default`
+Make your way to Meshery and download the token by clicking the "Get Token" option in the dropdown menu under your User Account avatar.
```
Comment (kushthedude) [inline docs/pages/installation/gke.md:9]: Same like above

File: docs/pages/installation/gke.md [hunk 5]
```diff
@@ -5,19 +5,31 @@ permalink: installation/gke
...
-Having run this script, supply the generated file `config-cluster-admin-sa-gke-default.yaml` in your Meshery settings page.
+Follow the below mentioned steps to set up manually:
+
+You may perform the steps outlined under [Managed Kubernetes](#managedk8s), following by hand or run the [generate_kubeconfig_gke.sh](./generate_kubeconfig_gke.sh) shell script using the desired ServiceAccount name and Namespace arguments, like so:
+
+`./generate_kubeconfig_gke.sh cluster-admin-sa-gke default`
 
-## Configuration
-This is where you configure your settings on the adaptor(Istio etc) and other things 
+Having run this script, supply the generated file config-cluster-admin-sa-gke-default.yaml in your Meshery settings page.
```
Comment (kushthedude) [inline docs/pages/installation/gke.md:26]: Can this be made a little more formal & detailed? Like : 
- After running this script, start the meshery server & navigate to `localhost:3000/settings` and provide the generated file xyz under Kube-config.

Comment (leecalcote) [reply docs/pages/installation/gke.md:26]: Good suggestion. Acknowledging that most Meshery users will have either deployed Meshery locally at `localhost:9081` or within a Kubernetes cluster and exposed access to it (which could be any IP address and port), perhaps, we should be unspecific in our reference to Meshery's location (URL).

**Comment by leecalcote** [comment]
Reviewing open PRs. This one is still pending an update.

File: docs/pages/installation/gke.md [hunk 6]
```diff
@@ -6,8 +6,9 @@ permalink: installation/gke
 
 # Quick Start with Google Kubernetes Engine (GKE)
 
-Make your way to Meshery and download the token by clicking the "Get Token" option in the dropdown menu under your User Account avatar.
-<br>Utilize the token to run the following command:
+- Make your way to meshery.ui, login with your user details and head over to the local port, found at <code>localhost:3000</code>.
```
Comment (leecalcote) [inline docs/pages/installation/gke.md:9]: "Navigate" more appropriate than "make your way".
"Meshery UI" more appropriate than "meshery.ui"
Avoid run-on sentence by placing a period after "In your web browser, navigate to Meshery UI being sure to include a port number as appropriate (e.g. `http://localhost:9081`)".
"Download an authentication token by..."

Don't use port 3000 as that is the port used for developers/contributors working on Meshery.

File: docs/pages/installation/eks.md [hunk 7]
```diff
@@ -6,10 +6,24 @@ permalink: installation/eks
...
-## Managed Kubernetes
+- Make your way to meshery.ui, login with your user details and head over to the local port, found at <code>localhost:3000</code>.
+- Download the token by clicking the "Get Token" option in the dropdown menu under your User Account avatar.
+- Utilize the token to run the following command:
+
+| command           | flag                | function                                                     | Usage                     |
+|:------------------|:-------------------:|:-------------------------------------------------------------|:--------------------------|
+|                   | --system config     | configures Meshery with the kubeconfig, generated with the help of user details, to provide cluster access for public clouds(EKS). | `mesheryctl system config gke --token "PATH TO TOKEN"` |
+
+Once configured, head over to the Quick Start Guide and continue with the steps outlined for <a href = "https://github.com/layer5io/meshery/blob/d75b6eec23021a1d8d3a38ec890fb10d44af3a35/docs/pages/installation/eks.md#managed-kubernetes"> EKS </a>.
```
Comment (leecalcote) [inline docs/pages/installation/eks.md:17]: Use a relative URL here.

File: docs/pages/installation/eks.md [hunk 8]
```diff
@@ -6,10 +6,24 @@ permalink: installation/eks
...
+
+## **Manual Configuration**
+
+Follow the below mentioned steps to set up manually:
+
+### **Managed Kubernetes**
 In order to run Meshery in a managed Kubernetes environment, you will need to assign an existing `ServiceAccount` or create a new `ServiceAccount`:
 
-Note: Make sure you are able to access EKS with kubectl by following the <a href="https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html" target="_blank"> EKS guide</a>
+Note: Make sure you are able to access EKS with kubectl by following the <a href="https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html" target="_blank"> EKS guide </a>
```
Comment (leecalcote) [inline docs/pages/installation/eks.md:26]: Put kubectl in backticks like so: `kubectl`

File: docs/pages/installation/eks.md (hunk 7 above)
Comment (leecalcote) [reply docs/pages/installation/eks.md:17]: Use markdown where possible, not HTML tags.

File: docs/pages/installation/eks.md [hunk 9]
```diff
@@ -6,15 +6,15 @@ permalink: installation/eks
 
 # Quick Start with Amazon Elastic Kubernetes Service (EKS)
 
-- Make your way to meshery.ui, login with your user details and head over to the local port, found at <code>localhost:3000</code>.
+- Navigate to the Meshery UI, login with your user details and head over to the local port, found at `localhost:9081.`
```
Comment (leecalcote) [inline docs/pages/installation/eks.md:9]: The location needs to be an example. localhost:9081 may or may not be where Meshery is deployed.

File: docs/pages/installation/eks.md [hunk 10]
```diff
@@ -23,7 +23,7 @@ Follow the below mentioned steps to set up manually:
 ### **Managed Kubernetes**
 In order to run Meshery in a managed Kubernetes environment, you will need to assign an existing `ServiceAccount` or create a new `ServiceAccount`:
 
-Note: Make sure you are able to access EKS with kubectl by following the <a href="https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html" target="_blank"> EKS guide </a>
+Note: Make sure you are able to access EKS with `kubectl` by following the <a href="https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html" target="_blank"> EKS guide </a>
```
Comment (leecalcote) [inline docs/pages/installation/eks.md:26]: Thanks for this. Please update the second URL to use markdown as well.

File: docs/pages/installation/gke.md [hunk 11]
```diff
@@ -24,8 +24,8 @@ You may perform the steps outlined under [Managed Kubernetes](#managedk8s), foll
 
 `./generate_kubeconfig_gke.sh cluster-admin-sa-gke default`
 
-Having run this script, fire up the meshery server and head over to the local port, usually found at <code>localhost:3000/settings</code> or <code>localhost:9081/settings</code>. This is where you configure your settings on the adaptor(Istio etc).
-<br>Supply the generated file <code><b> config-cluster-admin-sa-gke-default.yaml </b></code> under kube-config.
+Having run this script, fire up the meshery server and head over to the local port, usually found at `localhost:9081/settings`. This is where you configure your settings on the adaptor(Istio etc).
```
Comment (leecalcote) [inline docs/pages/installation/gke.md:27]: Let's strive to use more direct language. The colloquial language is excellent for blog posts, where an individual's voice can be heard. Example:


"Once the script is complete, you may proceed to start Meshery with the GKE-compatible configuration by executing `mesheryctl system start`."

You can remove this sentence: "This is where you configure your settings on the adaptor(Istio etc)."

File: docs/pages/installation/eks.md (hunk 2 above)
Comment (leecalcote) [reply docs/pages/installation/eks.md:16]: Let's go ahead and include this final instruction.

File: docs/pages/installation/gke.md (hunk 5 above)
Comment (leecalcote) [reply docs/pages/installation/gke.md]: @chandrashritii a reminder that the hostname and/or port number may change from deployment to deployment.

File: docs/pages/installation/gke.md (hunk 11 above)
Comment (chandrashritii) [reply docs/pages/installation/gke.md]: Made the advised changes. Thanks for pointing that out. I'll focus on using more direct language in the future. 
Please review @leecalcote.

File: docs/pages/installation/gke.md [hunk 12]
```diff
@@ -5,19 +5,32 @@ permalink: installation/gke
...
 
-## Managed Kubernetes
+### **Managed Kubernetes**
 In order to run Meshery in a managed Kubernetes environment, you will need to assign an existing `ServiceAccount` or create a new `ServiceAccount`:
 
 1. Create a `ServiceAccount` with `cluster-admin` role.
-1. Get secret name from `ServiceAccount`.
-1. Extract CA certificate and user token from the secret.
-1. Generate new kubeconfig yaml file to use as input to Meshery.
+2. Get secret name from `ServiceAccount`.
```
Comment (leecalcote) [inline docs/pages/installation/gke.md:34]: Using `1.` in markdown will auto-increment numbered lists.

File: docs/pages/installation/gke.md [hunk 13]
```diff
@@ -5,19 +5,32 @@ permalink: installation/gke
...
+
+`./generate_kubeconfig_gke.sh cluster-admin-sa-gke default`
 
-## Configuration
-This is where you configure your settings on the adaptor(Istio etc) and other things 
+Having run this script, fire up the meshery server and head over to the local port, usually found at `localhost:9081/settings`. This is where you configure your settings on the adaptor(Istio etc).
+<br>Supply the generated file `**config-cluster-admin-sa-gke-default.yaml**` under kube-config.
 
-## Managed Kubernetes
+### **Managed Kubernetes**
```
Comment (leecalcote) [inline docs/pages/installation/gke.md:30]: Let's avoid use of styling beyond the header style itself (`##`), if we can.

File: docs/pages/installation/gke.md (hunk 11 above)
Comment (leecalcote) [reply docs/pages/installation/gke.md]: The last sentence still needs to be removed. The "fire up" phrase is still being used. "head over to" is still being used. They don't need to configure their adapter. They need to run the command I provided to start Meshery.

File: docs/pages/installation/eks.md (hunk 2 above)
Comment (leecalcote) [reply docs/pages/installation/eks.md:16]: Inclusion of the table is out of place here. Keep it simple and just give the user the command they should run.

File: docs/pages/installation/gke.md [hunk 14]
```diff
@@ -5,19 +5,32 @@ permalink: installation/gke
...
+| command           | flag                | function                                                     | Usage                     |
+|:------------------|:-------------------:|:-------------------------------------------------------------|:--------------------------|
+|                   | --system config     | configures Meshery with the kubeconfig, generated with the help of user details, to provide cluster access for public clouds(GKE). | `mesheryctl system config gke --token "PATH TO TOKEN"` |
+
+Once configured, you can proceed to the Quick Start Guide and continue with the steps outlined for [GKE](/docs/installation/gke#manual-configuration).
+
+## **Manual Configuration**
 
-Having run this script, supply the generated file `config-cluster-admin-sa-gke-default.yaml` in your Meshery settings page.
+Follow the below mentioned steps to set up manually:
```
Comment (leecalcote) [inline docs/pages/installation/gke.md:21]: "Follow the below mentioned steps..." is grammatically incorrect.

File: docs/pages/installation/gke.md [hunk 15]
```diff
@@ -5,19 +5,32 @@ permalink: installation/gke
...
-`./generate_kubeconfig_gke.sh cluster-admin-sa-gke default`
+- Navigate to the Meshery UI, login with your user details and head over to the local port (e.g. `http://localhost:9081`).
+- Download the authentication token by clicking the "Get Token" option in the dropdown menu under your User Account avatar.
+- Utilize the token to run the following command:
+
+| command           | flag                | function                                                     | Usage                     |
+|:------------------|:-------------------:|:-------------------------------------------------------------|:--------------------------|
+|                   | --system config     | configures Meshery with the kubeconfig, generated with the help of user details, to provide cluster access for public clouds(GKE). | `mesheryctl system config gke --token "PATH TO TOKEN"` |
+
+Once configured, you can proceed to the Quick Start Guide and continue with the steps outlined for [GKE](/docs/installation/gke#manual-configuration).
```
Comment (leecalcote) [inline docs/pages/installation/gke.md:17]: A link to the Quick Start guide is needed, not a link back to this same document.

**Comment by leecalcote** [comment]
> I personally prefer `1,2,3`, they are more intuitive to read when someone is reading the source code. I say this is good to go @leecalcote

Like thoughtful variable names, yes, literal use of the number as a string is easier to read. Where this becomes a maintenance issue is when you need to rearrange the list or add/subtract a step, you have to then rework each of the literal numbers.

**Pull request merged**